quickstart/101-attestation-provider/variables.tf

```diff
@@ -1,9 +1,11 @@
-variable "resource_group_name" {
-  default = "myResourceGroup"
+variable "resource_group_name_prefix" {
+  default       = "rg"
+  description   = "Prefix of the resource group name that's combined with a random ID so name is unique in your Azure subscription."
 }
 
 variable "resource_group_location" {
   default = "eastus"
+  description   = "Location of the resource group."
```

Comment:
Can we run `terraform fmt` command for this directory please? The formatted line should be:

```hcl
variable "resource_group_location" {
  default     = "eastus"
  description = "Location of the resource group."
}
```